### PR TITLE
Fixed travis Python 3.6 build

### PR DESCRIPTION
**Pull request by abhishekpratapa**
Install virtualenv before running build-script.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,6 +34,9 @@ env:
 script:
   - free -m
 
+  # Hacky af: But to temporarily fix the travis build breaks
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then alias python=python3; alias pip=pip3; fi
```
Comment (TobyRoseman) [inline .travis.yml]: I would recommend doing `python=python3.6` and `pip=pip3.6`

Comment (abhishekpratapa) [reply .travis.yml]: Curious. I'll try that see if the build fails.

Comment (abhishekpratapa) [reply .travis.yml]: Updated the builds should work.

Comment (abhishekpratapa) [reply .travis.yml]: Updated. The builds should work.

**Comment by abhishekpratapa** [comment]
#2110

File: .travis.yml [hunk 2]
```diff
@@ -43,6 +43,9 @@ script:
   - tar xf cmake-3.13.4-Linux-x86_64.tar.gz
   - export PATH=${PWD}/cmake-3.13.4-Linux-x86_64/bin:${PATH}
 
+  # Use the correct virtualenv
```
Comment (znation) [inline .travis.yml]: Why does installing virtualenv suddenly cause it to use the correct virtualenv? A bit more of a comment here explaining the fix would be great.

Comment (abhishekpratapa) [reply .travis.yml]: Updated comment.

**Pull request merged**